lib/quicksilver-config-client.js

```diff
@@ -55,16 +60,16 @@ class QuicksilverConfigClient {
   }
 
   /**
-   * Check that configuration file define in .env as QS_CONFIG_PATH is accessable.
+   * Check that configuration path (qsConfigPath) value defined in .env as QS_CONFIG_PATH
+   * is a valid path.
    */
   configFileFound() {
     return new Promise((resolve) => {
-      fs.access(this.qsConfigPath, fs.F_OK, (err) => {
-        const fileFound = true;
+      fs.lstat(this.qsConfigPath, (err) => {
```

Comment:
You also need to check that this is a file. See [`stats.isFile()`](https://nodejs.org/api/fs.html#fs_class_fs_stats).